### PR TITLE
Privacy policy & overall enhancements

### PR DESCRIPTION
**Pull request by TomDijkema**
PR includes some overall enhancements based on smaller user stories, and adds an (empty) template for document pages like the Privacy Policy. Content for the Privacy Policy still needs to be delivered and pasted into the page.

Adds:
- License text to annotation form (CC0)
- Privacy Policy Page (missing content)
- Organisation logos (in JSON)

Modifies:
- Replaces MIDS bar with MIDS blocks for MIDS level indication

Resolves:
- Styling issues with the Annotation modal
- All Eslint notifications, except the one about including all variables used in an useEffect hook in the dependency array.
Not a fan of that because it drastically increases the amount of rerenders in the view.
This may be something to look at in the future, but my gut feeling is that it already is becoming quite smooth with the current stuff.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
 REACT_APP_PROXY_URL = https://sandbox.dissco.tech
-REACT_APP_AXIOS_URL = https://sandbox.dissco.tech/api/v1
+REACT_APP_AXIOS_URL = http://localhost:3000/api/v1
```
Comment (samleeflang) [inline .env:2]: Is this going to work?

File: src/sources/organisationLogos.json [hunk 2]
```diff
@@ -0,0 +1,461 @@
+{
+    "02gfc7t72": {
```
Comment (samleeflang) [inline src/sources/organisationLogos.json:2]: Mooi! Ben benieuwd of dit allemaal goed gaat met de ROR's, weet niet of ik altijd de juist gebruik.

File: src/components/specimen/Specimen.tsx [hunk 3]
```diff
@@ -85,25 +82,21 @@ const Specimen = () => {
...
-
-    /* Function for checking Specimen Annotations */
-    const CheckAnnotations = (specimen: SpecimenType) => {
-        GetSpecimenAnnotations(specimen.id).then((annotations) => {
-            if (annotations) {
-                dispatch(setSpecimenAnnotations(annotations));
-            }
-        });
-    }
+        // if (Array.isArray(annotateTarget.annotations) &&
```
Comment (samleeflang) [inline src/components/specimen/Specimen.tsx:90]: Is this commented piece needed?

**Review by samleeflang — COMMENTED** [review]
Looking great, lots of nice changes and lib updates

File: .env (hunk 1 above)
Comment (TomDijkema) [reply .env]: No, accidentally pushed the development variables, new commit resets them to production

File: src/components/specimen/Specimen.tsx (hunk 3 above)
Comment (TomDijkema) [reply src/components/specimen/Specimen.tsx]: I recall I commented this, but not sure, anyways uncommented this code, will look if things are working fine when everything comes together

**Review by samleeflang — APPROVED** [review]

**Pull request merged**